### PR TITLE
Css work

### PR DESCRIPTION
**Pull request by AnielaMW**

### CODE REVIEW

File: app/assets/stylesheets/dance.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#dance_comment_list li {
```
[the remark this answers is not in the document]
Comment (itiswicked) [reply app/assets/stylesheets/dance.scss:1]: This is true! In general you want to use classes for sections of the page, and if you need to, use ids for records in a list:

```
<%= @cars.each do |car| %>
  <div id="car-<%= car.id %>"><%= car.make %></div>
<% end %>
```

File: app/assets/stylesheets/dance_comments.scss [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  border: 3px solid black;
+  position: relative;
+}
+
+#comment_box p {
+  padding-left: 5px;
+}
+
+#comment_box footer ul {
+  position: absolute;
```
[the remark this answers is not in the document]
Comment (itiswicked) [reply app/assets/stylesheets/dance_comments.scss:13]: Ignore this one, its too particular

File: app/assets/stylesheets/dance_comments.scss [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  margin: 5px;
+  border: 3px solid black;
+  position: relative;
+}
+
+#comment_box p {
+  padding-left: 5px;
+}
+
+#comment_box footer ul {
```
[the remark this answers is not in the document]
Comment (itiswicked) [reply app/assets/stylesheets/dance_comments.scss:12]: `Selector should have depth of applicability no greater than 2, but was 3`

This rule kind of makes sense. I think a good compromise here is to use scss nested selectors:

```
#comment_box {
  // styles
  p {
    // styles
  }
  
  footer ul {
    // styles
  }
}
```

File: app/assets/stylesheets/dance_comments.scss [hunk 4]
```diff
@@ -0,0 +1,22 @@
+#comment_box {
+  border: 3px solid #000000;
```
[the remark this answers is not in the document]
Comment (itiswicked) [reply app/assets/stylesheets/dance_comments.scss:2]: This rule makes sense, especially if you are repeating colors throughout the app to give it a theme.
```
// colors.scss
$color-black: #000000;

// other-file.scss
@import 'colors';

.thing {
 color: $color-black;
}
```

File: app/assets/stylesheets/dance.scss (hunk 1 above)
[the remark this answers is not in the document]
Comment (itiswicked) [reply app/assets/stylesheets/dance.scss:1]: Just to expand on this topic a little more, typically classes and ids nowadays are not used for their intended purpose.

classes are for targeting with CSS, ids are used for targeting with JS, although ids can be used sparingly to style unique elements on a page in which using a class would not work

**Comment by itiswicked** [comment]
nice!
I would see if you can just restructure the CSS a little bit to take advantage of some Sass/ scss conventions.

I like the way you have structured the files so far. Eventually when you get a lot of CSS, it might make sense to structure the css files in a very similar fashion to your views files:

```
|_ pages
  |_ dances
    |_ show.scss
    |_ index.scss
  |_ other_resource
    |_ index.scss
```

**Comment by itiswicked** [comment]
Sorry about the close! Pressed the wrong button

This is all set to merge.

File: app/assets/stylesheets/dance_comments.scss (hunk 3 above)
[the remark this answers is not in the document]
Comment (AnielaMW) [reply app/assets/stylesheets/dance_comments.scss:12]: I had forgotten about nested selectors. I feel like I'm having to reteach myself everything as I move from html to css to javascript to AJAX and back to basic rails/ruby. If I'm out of one for too long I need to refresh on it to be able to start anything new.

**Pull request merged**